openvpn/version.go

```diff
@@ -0,0 +1,38 @@
+package openvpn
+
+import (
+	"os/exec"
+	"syscall"
+)
+
+//Version function checks that openvpn is available, given path to openvpn binary
+func Version(openvpnBinary string) error {
+
+	process := NewProcess(openvpnBinary, "[openvpn binary check]")
+	if err := process.Start([]string{"--version"}); err != nil {
+		return err
+	}
+	cmdResult := process.Wait()
+
+	exitCode, err := extractExitCodeFromCmdResult(cmdResult)
+	//openvpn returns exit code 1 in case of --version paramter, if anything else is returned - treat as error
```

Comment:
It will rarely will be something else on error than '1', in theory we should parse and look for version.. but prob, kad skip this for now.. :/

Reply:
That's the idea of this PR - to check that path to openvpn was correct. Exact openvpn version maching and even freeze is out of scope for this PR and I not sure that its needed.
This exitCode == 1 test is simply to disable Error messages that executable exited with status non-zero which does openvpn given version parameter